proto/priest.proto

```diff
@@ -78,14 +80,17 @@ message SmitePriest {
 	  
       RotationType rotation_type = 1;
       bool use_dev_plague = 3; // only undead
-      bool use_starshards = 4; // only nelf3
+      bool use_starshards = 4; // only nelf
+      bool use_mind_blast = 5;
+      bool use_SW_death = 6;
```

Comment:
If you don't mind lets name this field 'use_shadow_word_death'. I know its a bit verbose but the proto is the main interface so its better to be clear. Also no capital letters in field names, for consistency.